DISCOVER/07_code_of_conduct.md

```diff
@@ -69,19 +69,18 @@ Make sure that **everyone involved in your conference/event** is aware that the
 **Why You Need a Code of Conduct**
 - [Why to have a CoC and how to make one](http://incisive.nu/2014/codes-of-conduct/)
 - [Why conferences need a code of conduct](https://jacobian.org/writing/codes-of-conduct/)
-- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)
+<!-- Citation: Rachel Nabors, "You Literally Cannot Pay Me to Speak Without a Code of Conduct," published September 1, 2015. Available at: http://rachelnabors.com/2015/09/01/code-of-conduct/ (Currently unavailable). -->
```

Comment:
Same comment as above, would recommend we use[ markdown footnotes](https://github.blog/changelog/2021-09-30-footnotes-now-supported-in-markdown-fields/) rather than HTML comments.